js/app/game.js

```diff
@@ -263,6 +263,10 @@ define([
         }
 
         var changeScene = (sceneKey) => {
+            if(!this.scenes[sceneKey]) {
```

Comment:
Can you use the isSceneExist method here?

Reply:
I can't find that function

Reply:
https://github.com/The-Brains/ClickAndPointLib/blob/0623a4a363da10dac9f88ad41d083857c81aaea3/js/app/game.js#L181-L194